src/components/structures/LeftPanel.tsx

```diff
@@ -309,6 +309,22 @@ export default class LeftPanel extends React.Component<IProps, IState> {
                 }
                 break;
         }
+        const navAction = getKeyBindingsManager().getNavigationAction(ev);
+        switch (navAction) {
+            case KeyBindingAction.NextLandmark:
+                ev.stopPropagation();
+                ev.preventDefault();
+                (
+                    document.querySelector(".mx_RoomTile_selected") ||
+                    document.querySelector(".mx_RoomListHeader_contextMenuButton")
+                )?.focus();
+                break;
+            case KeyBindingAction.PreviousLandmark:
+                ev.stopPropagation();
+                ev.preventDefault();
+                document.querySelector(".mx_SpaceButton_active")?.focus();
```

Comment:
```suggestion
                document.querySelector<HTMLElement>(".mx_SpaceButton_active")?.focus();
```

Comment would be appreciated